packages/world/src/interfaces/IBaseWorld.sol

```diff
@@ -4,8 +4,7 @@ pragma solidity >=0.8.0;
 /* Autogenerated file. Do not edit manually. */
 
 import { IStore } from "@latticexyz/store/src/IStore.sol";
-import { IWorldKernel } from "../interfaces/IWorldKernel.sol";
-
+import { IWorldKernel } from "./IWorldKernel.sol";
```

Comment:
@dk1a I noticed the import here didn't resolve this path completely/correctly. It's not an issue as this import still works, but is a little more verbose than intended. Is this intentional?

Reply:
kind of intentional, the path just isn't resolved (for simplicity, because I can avoid it and it's a package import with a path rather than a proper path)